rocketpy/Flight.py

```diff
@@ -2645,327 +2081,165 @@ def angleOfAttack(self, interpolation="spline", extrapolation="natural"):
 
         # Calculate angle of attack and convert to degrees
         angleOfAttack = np.rad2deg(np.arccos(dotProductNormalized))
-        angleOfAttack = np.column_stack([self.x[:, 0], angleOfAttack])
-
-        # Convert to Function
-        angleOfAttack = Function(
-            angleOfAttack,
-            "Time (s)",
-            "Angle Of Attack (°)",
-            interpolation,
-            extrapolation,
-        )
+        angleOfAttack = np.column_stack([self.time, angleOfAttack])
 
         return angleOfAttack
 
-    # Stability and Control variables
-    # Angular velocities frequency response - Fourier Analysis
-    @cached_property
+    # Frequency response and stability variables
+    @funcify_method("Frequency (Hz)", "ω1 Fourier Amplitude", "spline", "zero")
     def omega1FrequencyResponse(self):
-        Fs = 100.0
-        # sampling rate
-        Ts = 1.0 / Fs
-        # sampling interval
-        t = np.arange(1, self.tFinal, Ts)  # time vector
-        y = self.w1(t)
-        y -= np.mean(y)
-        n = len(y)  # length of the signal
-        k = np.arange(n)
-        T = n / Fs
-        frq = k / T  # two sides frequency range
-        frq = frq[range(n // 2)]  # one side frequency range
-        Y = np.fft.fft(y) / n  # fft computing and normalization
-        Y = Y[range(n // 2)]
-        omega1FrequencyResponse = np.column_stack([frq, abs(Y)])
-        omega1FrequencyResponse = Function(
-            omega1FrequencyResponse, "Frequency (Hz)", "Omega 1 Angle Fourier Amplitude"
+        """Angular velocity 1 frequency response as a rocketpy.Function of frequency,
+        as the rocket leaves the launch rail for 5 seconds of flight."""
+        return self.w1.toFrequencyDomain(
+            self.outOfRailTime, self.outOfRailTime + 5, 100
```

Comment:
Defining the upper time bound to be just 5 seconds after the lower bound is what is making the weird graph shown in `Flight.allInfo()` that @Gui-FernandesBR  showed. This makes senses makes senses because smaller time frames will make the fft more 'broad' (or whatever the correct term is)

So the graph is right, but I don't know if having this is very useful compared to what we used before.

Just to exemplify:

**omega1FrequencyResponse in a 5 sec window:**

![Figure 12](https://user-images.githubusercontent.com/69485049/202884916-aa73972a-ca29-471f-8c33-5205bfa2fa3c.png)

**With xlim to 5hz:**

![Figure 14](https://user-images.githubusercontent.com/69485049/202885159-f377300e-f2dc-4780-9faf-775f5c91b5fa.png)

**omega1FrequencyResponse until Flight.tFinal (how it was before):**

![Figure 13](https://user-images.githubusercontent.com/69485049/202884983-e23343bc-9a16-4e79-ac01-cb05674052a1.png)

**With xlim to 5hz:**

![Figure 15](https://user-images.githubusercontent.com/69485049/202885096-df1a30e3-aa1e-4ede-bb01-0d92068a54be.png)


I think it makes more sense to look at the frequency response of the entire flight in this case. Or maybe just make the window bigger than 5 seconds

What do you guys think @giovaniceotto @Gui-FernandesBR ?